app/config/default_parameters.yml

```diff
@@ -139,9 +139,12 @@ parameters:
     # form builder
     ezsettings.default.form_builder.forms_location_id: 239
 
+    # Location Id of administrator users group
+    app.administrator_users.location_ids: 13
+
     # Location Ids of allowed user groups for viewing Articles with premium content
     # 12 - members, 13 administrator_users, 14 - editors
-    app.premium_content.allowed_user_groups.location_ids: [12, 13, 14]
+    app.premium_content.allowed_user_groups.location_ids: [12, '%app.administrator_users.location_ids%', 14]
```

Comment:
```suggestion
    app.premium_content.allowed_user_groups.location_ids: [12, '%app.administrator_group.location_id%', 14]
```

Reply:
Side note: I think the new design does not include the premium content. I think we should remove those parameters.